components/service/glean/src/main/java/mozilla/components/service/glean/EventMetricType.kt

```diff
@@ -24,12 +24,13 @@ internal const val MAX_LENGTH_OBJECT_ID = 20
  * data and making sure that limits are enforced.
  */
 data class EventMetricType(
-    override val applicationProperty: Boolean,
-    override val disabled: Boolean,
-    override val group: String,
+    override val applicationProperty: Boolean = false,
```

Comment:
Does it make sense for these defaults to be in `CommonMetricData` instead?

Reply:
If I do that, I get the message "Property initializers are not allowed on interfaces".

An alternative might be to not skip defaults in the `glean_parser` generator, but always specify them explicitly.  Then at least the defaults are only defined in one place (glean_parser).  I don't know if there are code size implications to that, though.

Reply:
Mh, @pocmo , do you have any advice for this? I would be for skipping defaults and always generate named arguments for all the parameters. Do you think this would affect the code size too much?